test/cutting/qpd/test_qpd.py

```diff
@@ -283,6 +286,6 @@ def test_qpdbasis_from_gate_unique_maps(
     def test_supported_gates(self):
         gates = supported_gates()
         self.assertEqual(
-            {"rxx", "ryy", "rzz", "crx", "cry", "crz", "cx", "cy", "cz", "ch", "csx"},
+            {"rxx", "ryy", "rzz", "crx", "cry", "crz", "cx", "cy", "cz", "ch", "csx", "ecr"},
```

Comment:
```suggestion
            {   
                "rxx",
                "ryy",
                "rzz",
                "crx",
                "cry",
                "crz",
                "cx",
                "cy",
                "cz",
                "ch",
                "csx",
                "ecr",
            },
```

I think this will fix your black error, but you'd probably rather just do it yourself to be 100% sure